recipes/sanic/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sanic" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "f70fbd2483e372494d023a08259ffbeb757b3bce10bbf4a00b17a405de3ff322" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<35 or win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - httptools >=0.0.9
+    - ujson >=1.35
+    - uvloop >=0.5.3
+    - websockets >=3.2
+    - aiofiles >=0.3.0
+
+test:
+  imports:
+    - sanic
+
+about:
+  home: https://github.com/channelcat/sanic
+  summary: "Async Python 3.5+ web server that's written to go fast"
+  license: MIT
```

Comment:
You should [package the license manually](http://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually) and send a PR upstream adding it to `MANIFEST.in`.

Reply:
Ongoing https://github.com/channelcat/sanic/pull/981